lib/src/signed_video_h26x_sign.c

```diff
@@ -658,12 +609,17 @@ signed_video_get_nalu_to_prepend(signed_video_t *self,
     signed_video_nalu_to_prepend_t *nalu_to_prepend)
 {
```

Comment:
Note that this function provides the user with SEIs in order from newest to oldest. You need to make sure that is still the case.